src/chocolatey/infrastructure.app/services/WindowsFeatureService.cs

```diff
@@ -226,53 +227,53 @@ public IEnumerable<PackageResult> list_run(ChocolateyConfiguration config)
             return packageResults;
         }
 
-        public string build_args(ChocolateyConfiguration config, IDictionary<string, ExternalCommandArgument> argsDictionary)
+        private string BuildArguments(ChocolateyConfiguration config, IDictionary<string, ExternalCommandArgument> argsDictionary)

```

Comment:
We don't have to, it simply felt like it was easier to do now that I'm renaming everything anyway, rather than have two layers of obsoletion to contend with in future. Happy to remove if you feel like this isn't worth doing here.

Reply:
Given that we are going through all this work to do things "properly" with the renaming of all the things, it would seem like the wrong thing to do here, and we should follow the same process.  I know that it adds another layer onto what we have been doing, but it still feels like the right thing to do to me.